control/timeresp.py

```diff
@@ -65,12 +65,18 @@
 capability and better automatic time vector creation
 Date: June 2020
 
+Modified by Ilhan Polat to improve automatic time vector creation
```

Comment:
I think you should get the credit here as far as I am concerned. I really didn't do anything other than geeking out some years ago.

Reply:
Geeking out = valuable! : ) If you're ok with it, I would just say leave it in - I took credit elsewhere in the code for merging things together.